waitress/pantry/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from django.contrib.auth.models import User
 
 # Register your models here.
+from pantry.models import Pantry
```

Comment:
You didn't properly register the model here.

[Here's an example](https://github.com/andela/waitress/blob/master/waitress/app/admin.py#L36) of how to do it. I pulled it from the `admin.py` file in the waitress app.
Hope it helps.